earth2studio/models/px/dlesym.py

```diff
@@ -32,6 +33,7 @@
     OmegaConf = None
     earth2grid = None
 from earth2studio.models.auto import AutoModelMixin, Package
+from earth2studio.models.batch import batch_coords, batch_func
 from earth2studio.models.px.base import PrognosticModel
 from earth2studio.models.px.utils import PrognosticMixin
 from earth2studio.utils import check_extra_imports, handshake_coords, handshake_dim
```

Comment:
Does _ATMOS_VARIABLES below need to get updated now that the model computes the derived variables?

Reply:
No, that hardcode is just there for reference -- but this is the core variable set so it remains the same. Only `DLESyMLatLon` accepts the non-derived input vars, and it modifies the input coords accordingly